sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -1708,7 +1777,41 @@ void set(
         checkArgument(
             spec.getPreparedStatementSetter() != null, "withPreparedStatementSetter() is required");
       }
-      return input
+      PCollection<Iterable<T>> iterables;
```

Comment:
Can we share code between WriteWithResult and WriteVoid ? This introduces a significant amount of code duplication.

Reply:
+1

Reply:
done. Added a helper method.